src/lib/utils/time.ts

```diff
@@ -0,0 +1,64 @@
+export const millisecondsPerSecond = 1000
+export const secondsPerMinute = 60
+export const minutesPerHour = 60
+export const hoursPerDay = 24
+export const daysPerWeek = 7
+
+export const millisecondsPerMinute = millisecondsPerSecond * secondsPerMinute
+export const millisecondsPerHour = millisecondsPerMinute * minutesPerHour
+export const millisecondsPerDay = millisecondsPerHour * hoursPerDay
+export const millisecondsPerWeek = millisecondsPerDay * daysPerWeek
+export const previousSevenDays = millisecondsPerWeek
+export const lastThirtyDays = millisecondsPerDay * 30
+
+export const currentYear = new Date().getFullYear()
+
+export function getTimeSince(date: Date) {
+  const now = new Date()
+  return now.getTime() - date.getTime()
+}
+
+export function getTimeSinceToString(date: Date) {
```

Comment:
If you rename the other function, then this function could be called just `getTimeSince` again, which reads a bit better than the current name 😃

Reply:
Or just `timeSince` would be enough I think, actually.